src/components/app-bar/AppBar.svelte

```diff
@@ -87,3 +90,47 @@
 
 <LoadingModal />
 <LoginModal />
+
+<style>
+	.header {
+		position: relative;
+		display: flex;
+		justify-self: center;
+		justify-content: space-between;
+		align-items: end;
+		min-height: 80px;
+		padding: 8px;
+		width: 100%;
+		margin-top: 10px;
+	}
+	.first {
```

Comment:
```suggestion
	}
	
	.first {
```
stay consistent with new lines.